### DISCUSSION TITLE
1.0.0

### DISCUSSION THREAD

**Discussion by AstralVixen** [AstralVixen/GeForce-Infinity#1] [General]
### It's finally here!

After a long time of abandoning this project I recently came back to it, thanks to the people who starred and forked my repo! I noticed a bit late but it gave me a huge motivation boost and now after some big changes I feel like version 1.0.0 is ready!

As it is 1.0.0 it will have bugs and is nowhere closed to finished but it has it's core functionalities as of now.
The .exe and .zip files are the windows release files the rest are for Linux!

<hr /><em>This discussion was created from the release <a href='https://github.com/AstralVixen/GeForce-Infinity/releases/tag/1.0.0'>1.0.0</a>.</em>

**Comment by rootiest** [comment]
I just want to drop by to say: (maybe as some motivation to keep at it!)

I have tried several different official/unofficial geforce now apps and yours is the only one that works flawlessly on my Arch KDE Wayland system. 

A lot of the official ones struggled to capture the mouse properly (official flatpak version) or were overly-agressive about capturing keystrokes (official electron version blocked my volume knob from controlling volume, blocked screenshots from the desktop, etc)

I have had none of those issues with your app, and the extras like discord RPC are also great!

So just wanted to say thank you for making this and I hope you keep working on it! :heart:

**Reply by AstralVixen** [reply]
Oh damm this was unexpected. Thank you so much lol! 

This project is on a bit of a hibernation for now at least until I finish my 2 other projects & then imma come back to release an update. Many features are planned tho already.
